feat: split and export renovate presets (#1516)
AmadeusITGroup/otter: docs/renovate/readme.md

# Renovate Presets

> The Otter library provide a set of [Renovate Presets](https://docs.renovatebot.com/config-presets/) custom to facilitate the configuration of automated dependency updates.

Reviewer: ```suggestion
The Otter library provides a set of custom [Renovate Presets](https://docs.renovatebot.com/config-presets/) to facilitate the configuration of automated dependency updates.
```

## Usage

> To be able to benefice of the Renovate Presets exposed by the Otter Framework, you can add the configurations in the `extends` field in your `.renovaterc.json` file as described in the following example:

Reviewer: ```suggestion
To be able to benefit from the Renovate Presets exposed by the Otter Framework, you can add the configurations in the `extends` field in your `.renovaterc.json` file as described in the following example:
```

```json5
{
  "extends": [
    "config:base"
    "github>AmadeusITGroup/otter//tools/renovate/base",
    "github>AmadeusITGroup/otter//tools/renovate/otter-project"
  ],
  "packageRules": [
    // ...
  ]
}
```

> [!NOTE]
> The Otter Renovate presets are always prefixed with `github>AmadeusITGroup/otter//tools/renovate/` to target the specific configuration files hosted in the Otter repository.

## Available presets

- **base**: Base configuration recommended to an Otter base project.

> - **otter-project**: Configuration to used in a project to ensure, facilitate and regroup the upgrades of the Otter dependencies.

Reviewer: ```suggestion
- **otter-project**: Configuration to be used in a project to facilitate and regroup the upgrades of Otter dependencies.
```
